pkg/publicapi/endpoint/orchestrator/endpoint.go

```diff
@@ -34,7 +33,6 @@ func NewEndpoint(params EndpointParams) *Endpoint {
 	// JSON group
 	g := e.router.Group("/api/v1/orchestrator")
 	g.Use(middleware.SetContentType(echo.MIMEApplicationJSON))
-	g.Use(echo_middleware.CORS())
```

Comment:
why did we need to remove this? I don't have a problem with the change - just curious

Reply:
It's added at https://github.com/bacalhau-project/bacalhau/pull/3711/files#diff-19907c38cadb733516c5d0d2d452e8ef1dde593253b301251fec43f1a8a8d315 instead

Reply:
https://github.com/bacalhau-project/bacalhau/pull/3711/commits/134db3f2c8a082744a4b8232ba30a7cc54fe18db